### PR TITLE
dependency changes for hedis

### PR DESCRIPTION
**Pull request by vijaygupta18**

### CODE REVIEW

File: flake.nix [hunk 1]
```diff
@@ -57,6 +52,8 @@
             mysql-haskell.source = inputs.mysql-haskell;
             sequelize.source = inputs.sequelize;
             cereal.source = inputs.cereal;
+            euler-events-hs.source = inputs.euler-events-hs;
```
Comment (aravindgopall) [inline flake.nix:55]: is this change necessary?

File: flake.nix [hunk 2]
```diff
@@ -42,10 +41,6 @@
         packages.default = self'.packages.euler-hs;
         haskellProjects.default = {
           projectFlakeName = "euler-hs";
-          imports = [
```
[the remark this answers is not in the document]
Comment (vijaygupta18) [reply flake.nix:45]: it was there in previous file..will revert this change as its not needed
